doctr/models/recognition/transformer.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+# This module 'transformer.py' is 100% inspired from this Tensorflow tutorial:
+# https://www.tensorflow.org/text/tutorials/transformer
+
+
+from typing import Tuple
+
+import tensorflow as tf
+import numpy as np
+
+
+__all__ = ['Decoder', 'positional_encoding', 'create_look_ahead_mask', 'create_padding_mask']
+
+
+def get_angles(pos: np.array, i: np.array, d_model: int = 512) -> np.array:
+    angle_rates = 1 / np.power(10000, (2 * (i // 2)) / np.float32(d_model))
+    return pos * angle_rates
```

Comment:
Could you add a docstring? I must admit I'm a bit lost about what it's supposed to be doing :sweat_smile:

Reply:
(even a one-liner docstring without argument definition)